python/pylibcudf/pylibcudf/io/json.pyx

```diff
@@ -81,6 +83,36 @@ cdef json_reader_options _setup_json_reader_options(
     opts.enable_keep_quotes(keep_quotes)
     opts.enable_mixed_types_as_string(mixed_types_as_string)
     opts.enable_prune_columns(prune_columns)
+    if extra_parameters is not None:
```

Comment:
Can you add a comment explaining that these options are in flux and they've been added to help test the Spark JSON reader?